standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/columnstats/merge/DecimalColumnStatsMergerTest.java

```diff
@@ -37,27 +40,32 @@
 
 @Category(MetastoreUnitTest.class)
 public class DecimalColumnStatsMergerTest {
-
-  private static final Decimal DECIMAL_1 = DecimalUtils.getDecimal(1, 0);
-  private static final Decimal DECIMAL_3 = DecimalUtils.getDecimal(3, 0);
-  private static final Decimal DECIMAL_5 = DecimalUtils.getDecimal(5, 0);
-  private static final Decimal DECIMAL_20 = DecimalUtils.getDecimal(2, 1);
```

Comment:
Does the method `DecimalUtils.getDecimal` used elsewhere? We can drop it if not.

Reply:
The class DecimalUtils is part of `org.apache.hadoop.hive.metastore.api.utils`. I'm not familiar with the usage of the API. Is it ok to delete a public static method from the public class `DecimalUtils`, @kasakrisz?

Reply:
Thanks for checking, I'll remove it.